chart/templates/iq-server-deployment.yaml

```diff
@@ -218,7 +218,15 @@ spec:
       initContainers:
         - name: {{ .Release.Name }}-set-iq-persistence-ownership
           image: busybox:1.28
-          command: ["sh", "-c", "chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}"]
+          command:
+            - "sh"
+            - "-c"
+            - >
+              {{- if .Values.customInitCommand -}}
```

Comment:
need an description for this in https://github.com/sonatype/nexus-iq-server-ha/blob/main/chart/README.md#chart-configuration-options

Reply:
Commit: 55959be
Just to clarify, I could say that the default value (default column in the table) is the `chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}` however it is not really the default value of the `pvOwnershipOverride` property unless I put the default command as value for it in the Values.yaml. What do you think?

Reply:
> Commit: [55959be](https://github.com/sonatype/nexus-iq-server-ha/commit/55959be5d05ddd1910789158fa7568ec253a9820)
> Just to clarify, I could say that the default value (default column in the table) is the `chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}` however it is not really the default value of the `pvOwnershipOverride` property unless I put the default command as value for it in the `Values.yaml`. What do you think?

Let's put that as the default value in Values.yaml

>need an description for this in https://github.com/sonatype/nexus-iq-server-ha/blob/main/chart/README.md#chart-configuration-options
Also, I take my previous comment back. Let's not document this and keep it as an undocumented feature for now.